ARImageTrackingExamples/ARImageTrackingExamples/View Controllers/SolidCardViewController.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import UIKit
+import ARKit
+
+class SolidCardViewController: UIViewController, ARExperimentSessionHandler {
+
+    @IBOutlet var sceneView: ARSCNView!
+    private let arSessionDelegate = ARExperimentSession()
+    private let arViewModel = ARViewModel()
+
+    override func viewDidLoad() {
+        sceneView.delegate = self
```

Comment:
you need `super.viewDidLoad()` here, else bad things can happen :D

Reply:
Thanks for the catch 😄